Source/WebCore/animation/KeyframeEffect.cpp

```diff
@@ -2167,7 +2167,7 @@ std::optional<KeyframeEffect::RecomputationReason> KeyframeEffect::recomputeKeyf
         return { };
 
     auto fontSizeChanged = [&]() {
-        return previousUnanimatedStyle && previousUnanimatedStyle->computedFontSize() != unanimatedStyle.computedFontSize();
+        return previousUnanimatedStyle && previousUnanimatedStyle->specifiedFontSize() != unanimatedStyle.specifiedFontSize();
```

Comment:
But it seems this can change for other reasons as well, right? Say the root element font-size is 12px and you change a random element from 12px to 1rem, now the specified value will have changed, but the computed value has not. This seems quite observable so this should be tested and standardized in some fashion.

Perhaps it's already tested, just with the wrong expectations, but it seems the web standard piece is still missing at least.

Reply:
I think this change is indeed incorrect, the existing code was fine. I'm reverting it in a newer patch.